lib/fizzy/execute.cpp

```diff
@@ -367,6 +367,8 @@ inline void convert(OperandStack& stack) noexcept
 template <typename SrcT, typename DstT>
 inline void reinterpret(OperandStack& stack) noexcept
 {
+    static_assert(std::is_integral_v<SrcT> == std::is_floating_point_v<DstT> ||
```

Comment:
Not sure this one is useful, but it is not apparent based on comments that this is only used to convert between floating point <-> integral.